### PR TITLE
Kate/87485/add Turbos to the Trade types list

### PR DESCRIPTION
**Pull request by kate-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
- Add Turbos to the Trade types (without NEW! label). 
- Add Turbos to the Dropdown of available Trade types.
- Add Description for Turbos, which is shown when Info icon is clicked, move description for a separate component. For mobile version add info icons, which are always visible, as it's not possible to implement hover analog. 
- Add Icons for Turbos.

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/components/src/components/icon-trade-types/icon-trade-types.jsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const IconTradeTypes = ({ type, className, ...props }) => {
             return <Icon icon='IcTradetypeTicklow' className={className} color='brand' {...props} />;
         case 'turboslong':
             return <Icon icon='IcTradetypeMultdown' className={className} color='brand' {...props} />;
-        case 'turbosshort':
+        case 'turbosshot':
             return <Icon icon='IcTradetypeMultup' className={className} color='brand' {...props} />;
```
Comment (maryia-deriv) [inline packages/components/src/components/icon-trade-types/icon-trade-types.jsx:75]:
```suggestion
            return <Icon icon='IcTradetypeTurbosshort' className={className} color='brand' {...props} />;
```

File: packages/components/src/components/icon-trade-types/icon-trade-types.jsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ const IconTradeTypes = ({ type, className, ...props }) => {
             return <Icon icon='IcTradetypeTicklow' className={className} color='brand' {...props} />;
         case 'turboslong':
             return <Icon icon='IcTradetypeMultdown' className={className} color='brand' {...props} />;
```
Comment (maryia-deriv) [inline packages/components/src/components/icon-trade-types/icon-trade-types.jsx]:
```suggestion
            return <Icon icon='IcTradetypeTurboslong' className={className} color='brand' {...props} />;
```

File: packages/components/src/components/icon-trade-types/icon-trade-types.jsx (hunk 1 above)
Comment (maryia-deriv) [inline packages/components/src/components/icon-trade-types/icon-trade-types.jsx]:
```suggestion
        case 'turbosshort':
            return <Icon icon='IcTradetypeTurbosshort' className={className} color='brand' {...props} />;
```

File: packages/trader/src/Assets/Trading/Categories/turboslong-trade-description.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            text: localize(
+                'Predict the market direction, size and also how confident you are that the spot price will not touch the barrier.'
+            ),
+        },
+        {
+            type: 'paragraph',
+            text: localize(
+                'If you select “Long”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always above the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Long” contract if you think the market will grow strongly without falling.'
+            ),
+        },
```
Comment (maryia-deriv) [inline packages/trader/src/Assets/Trading/Categories/turboslong-trade-description.jsx:17]: if only 1 paragraph differs, we can reuse the component 🙂:
```suggestion
export const TurbosTradeDescription = (is_turboslong) => {
    const second_paragraph = is_turboslong
            ? localize(
                'If you select “Long”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always above the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Long” contract if you think the market will grow strongly without falling.'
            )
            : localize(
                'If you select “Short”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always below the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Short” contract if you think the market will decay strongly without rising.'
            );
    const content = [
        {
            type: 'paragraph',
            text: localize(
                'Predict the market direction, size and also how confident you are that the spot price will not touch the barrier.'
            ),
        },
        {
            type: 'paragraph',
            text: second_paragraph,
        },
```

Comment (kate-deriv) [reply packages/trader/src/Assets/Trading/Categories/turboslong-trade-description.jsx:17]: If the difference between final text will be minor, I'll reuse that component.

**Review by maryia-deriv — APPROVED** [review]
👍🏻

**Pull request merged**